.github/workflows/build-and-push.yml

```diff
@@ -64,6 +64,11 @@ jobs:
           TOXENV: ${{ matrix.toxenv }}
         run: |
           docker run --rm --platform linux/${{ matrix.arch }} -e DNF_INSTALL="libffi-devel pkgconfig(libgit2) /usr/bin/cowsay" fedorapython/fedora-python-tox:${{ matrix.arch }} sh -c "/run_tests.sh; pip install -I --no-deps --compile --no-binary :all: cffi pygit2~=1.14.0 && cowsay DONE"
+      - name: Test dnf install with version specifiers
+        env:
+          TOXENV: ${{ matrix.toxenv }}
+        run: |
+          docker run --rm --platform linux/${{ matrix.arch }} -e DNF_INSTALL="'pytest > 7' 'cowsay > 2'" fedorapython/fedora-python-tox:${{ matrix.arch }} sh -c "/run_tests.sh; cowsay DONE"
```

Comment:
I'd fold this to the previous test to save some time. But not a hard requirement.